<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>637</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Task Managment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget_mainTabs">
      <property name="styleSheet">
       <string notr="true">q
rgb(241, 255, 231)</string>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_tasks">
       <attribute name="title">
        <string>Tasks and Events</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_addAndSearch">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_addTask">
             <property name="styleSheet">
              <string notr="true">#pushButton_addTask {
    background-color: #3498db;
    color: white;
    font-size: 14px;
}
#pushButton_addTask:hover {
	background-color: rgb(174, 223, 255);
    color: white;
    font-size: 14px;

}</string>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_searchTask">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_searchTask">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="placeholderText">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_sort1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_sort2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollArea" name="scrollArea_tasks">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollArea_tasks_content">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1071</width>
             <height>453</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_calender">
       <attribute name="title">
        <string>Calender</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCalendarWidget" name="calendarWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_study">
       <attribute name="title">
        <string>Study Techniques</string>
       </attribute>
       <widget class="QComboBox" name="comboBox_techniques">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>251</width>
          <height>21</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Your Own Time Blocking</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pomodoro Technique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>52-17 Technique</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>The 45-15 Method</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="copmletion_label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose your technique:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_study">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>30</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_study {
    background-color: #3498db;
    color: white;
    font-size: 14px;
}
#pushButton_study:hover {
	background-color: rgb(0, 85, 255);
    color: white;
    font-size: 14px;

}</string>
        </property>
        <property name="text">
         <string>START</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>19</x>
          <y>79</y>
          <width>551</width>
          <height>401</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTextBrowser" name="TextBrowser_display">
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="pushButton_study_2">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_study_2 {
    background-color: #3498db;
    color: white;
    font-size: 14px;
}
#pushButton_study_2:hover {
	background-color: rgb(0, 85, 255);
    color: white;
    font-size: 14px;

}</string>
        </property>
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_study_3">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">#pushButton_study_3 {
    background-color: #3498db;
    color: white;
    font-size: 14px;
}
#pushButton_study_3:hover {
	background-color: rgb(0, 85, 255);
    color: white;
    font-size: 14px;

}</string>
        </property>
        <property name="text">
         <string>RESET</string>
        </property>
       </widget>
       <widget class="QLabel" name="copmletion_label_4">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>250</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Your Progress:</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="completion_bar_3">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>250</y>
          <width>261</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="copmletion_label_5">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>70</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>TIMER:</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="doneNum_3">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>60</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="copmletion_label_6">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>170</y>
          <width>51</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>850</x>
          <y>170</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Studying&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <zorder>copmletion_label_3</zorder>
       <zorder>pushButton_study</zorder>
       <zorder>verticalLayoutWidget</zorder>
       <zorder>pushButton_study_2</zorder>
       <zorder>pushButton_study_3</zorder>
       <zorder>copmletion_label_4</zorder>
       <zorder>completion_bar_3</zorder>
       <zorder>copmletion_label_5</zorder>
       <zorder>doneNum_3</zorder>
       <zorder>copmletion_label_6</zorder>
       <zorder>textBrowser_2</zorder>
       <zorder>comboBox_techniques</zorder>
      </widget>
      <widget class="QWidget" name="tab_insights">
       <attribute name="title">
        <string>Insights and Analytics</string>
       </attribute>
       <widget class="QGroupBox" name="myDay_data">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>491</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>My Day</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QProgressBar" name="completion_bar">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>40</y>
           <width>261</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="copmletion_label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Completion Percentage:</string>
         </property>
        </widget>
        <widget class="QLabel" name="done">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Total Tasks Done:</string>
         </property>
        </widget>
        <widget class="QLabel" name="left">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Tasks Left:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="doneNum">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>80</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLCDNumber" name="leftNum">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>120</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="Line" name="analytics_tab_Vdivider">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>20</width>
          <height>181</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QGroupBox" name="prev_month">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>10</y>
          <width>491</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>Previous Month</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QProgressBar" name="completion_bar_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>40</y>
           <width>261</width>
           <height>23</height>
          </rect>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QLabel" name="copmletion_label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Completion Percentage:</string>
         </property>
        </widget>
        <widget class="QLabel" name="done_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Total Tasks Done:</string>
         </property>
        </widget>
        <widget class="QLabel" name="left_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>151</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Tasks Left:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="doneNum_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>80</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLCDNumber" name="leftNum_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>120</y>
           <width>64</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="myDay_data_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>200</y>
          <width>1061</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>Charts</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QLabel" name="chart1_2">
         <property name="geometry">
          <rect>
           <x>570</x>
           <y>10</y>
           <width>401</width>
           <height>311</height>
          </rect>
         </property>
         <property name="text">
          <string>Chart generated from data</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="chart1_3">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>401</width>
           <height>311</height>
          </rect>
         </property>
         <property name="text">
          <string>Chart generated from data</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1115</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionSave_Data"/>
    <addaction name="actionBack_Up_Data"/>
    <addaction name="actionRestore_Data"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionBack_Up_Data">
   <property name="text">
    <string>Back-Up Data</string>
   </property>
  </action>
  <action name="actionRestore_Data">
   <property name="text">
    <string>Restore Data</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
